lib/config.js

```diff
@@ -307,7 +307,16 @@ config.start = function start() {
     // Wait until metrics has started before starting cluster control, because
     // the workers need to know what ephemeral port the statsd server is
     // listening on in order to send it metrics.
-    return control.start(config);
+    debug('cluster-control size: %j', config.size);
+    var size = config.size < 0 ? {} : { size: config.size };
+    var c = control.start(size);
```

Comment:
The return value was intended to be removed, but I looked closer, and realized the return value isn't used.